### DISCUSSION TITLE
dynamic images as background

### DISCUSSION THREAD

**Discussion by dHarshMakwana** [tailwindlabs/tailwindcss#11848] [Help]
I am trying to make some cards with background i am trying to map it,
and i using image as arbitary value, i am getting error when i run it something related to global.css

**Comment by wongjn** [comment]
Sounds like you might be trying to use string interpolation/concatenation and as per [the documentation](https://tailwindcss.com/docs/content-configuration#dynamic-class-names):

> The most important implication of how Tailwind extracts class names is that it will only find classes that exist as _complete unbroken strings_ in your source files.
>
> If you use string interpolation or concatenate partial class names together, Tailwind will not find them and therefore will not generate the corresponding CSS:
>
> **Don’t construct class names dynamically**
>
> ```vue
> <div class="text-{{ error ? 'red' : 'green' }}-600"></div>
> ```
> In the example above, the strings `text-red-600` and `text-green-600` do not exist, so Tailwind will not generate those classes.
> Instead, make sure any class names you’re using exist in full:
>
> **Always use complete class names**
>
> ```vue
> <div class="{{ error ? 'text-red-600' : 'text-green-600' }}"></div>
> ```
